### PR TITLE
Bug 1706689: asset/manifests: internal api server URL for infrastructure 

### PR DESCRIPTION
**Pull request by abhinavdahiya**
Catch up to the api change introduced in openshift/api#308

### CODE REVIEW

File: cmd/openshift-install/create.go [hunk 1]
```diff
@@ -326,6 +326,7 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 	clusterVersionContext, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 
+	failing := configv1.ClusterStatusConditionType("Failing")
```
Comment (wking) [inline cmd/openshift-install/create.go]: Is this just until the CVO moves to Degraded?

Comment (abhinavdahiya) [reply cmd/openshift-install/create.go]: no plans for CVO to move to `Degraded`

Comment (wking) [reply cmd/openshift-install/create.go]: We're just going to translate operator Degradeds into CVO Failings forever?

Comment (abhinavdahiya) [reply cmd/openshift-install/create.go]: > no plans for CVO to move to `Degraded`
> We're just going to translate operator Degradeds into CVO Failings forever?
`forever` i can't say ;)
for now yes
`We're` the installer only cares about the fact that CVO reports failing, its only a consumer. the installer shouldn't care

**Comment by wking** [comment]
/lgtm

@deads2k thonks this might cause issues with in-cluster cert minting/rotating.  But if it gets through CI it's fine with me, we just need to check/fix rotation before cutting a release.

**Comment by wking** [comment]
/lgtm

with the ratchet too ;)

File: cmd/openshift-install/create.go (hunk 1 above)
Comment (eparis) [reply cmd/openshift-install/create.go]: every team was told to move. why didn't the cvo move?

**Comment by abhinavdahiya** [comment]
[e2e-aws](https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_installer/1718/pull-ci-openshift-installer-master-e2e-aws/5689)

failing console...
```
level=fatal msg="failed to initialize the cluster: Some cluster operators are still updating: authentication, console: timed out waiting for the condition"
```
/retest

**Comment by spadgett** [comment]
> [e2e-aws](https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_installer/1718/pull-ci-openshift-installer-master-e2e-aws/5689)
> 
> failing console...
> 
> ```
> level=fatal msg="failed to initialize the cluster: Some cluster operators are still updating: authentication, console: timed out waiting for the condition"
> ```

Looks like https://bugzilla.redhat.com/show_bug.cgi?id=1707510

**Comment by wking** [comment]
[e2e-aws-upgrade][1]:

```
fail [k8s.io/kubernetes/test/e2e/framework/service_util.go:589]: May  7 17:26:06.627: Timed out waiting for service "service-test" to have a load balancer
...
Failing tests:

[Disruptive] Cluster upgrade should maintain a functioning cluster [Feature:ClusterUpgrade] [Suite:openshift] [Serial]
```

I've reaped a few leaked resources, so these should go through now.

/retest

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1718/pull-ci-openshift-installer-master-e2e-aws-upgrade/273/build-log.txt

**Comment by wking** [comment]
/hold

We want this, but it's not a beta-blocker.  Putting the breaks on to let #1720 pass us in the merge queue.

**Pull request merged**